src/tooltip/index.es

```diff
@@ -1,21 +1,58 @@
-import React from 'react';
+import React, { useCallback, useRef, useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import clsx from 'clsx';
+import { createPopper } from '@popperjs/core';
 
 import { arrayToHash } from 'nebenan-helpers/lib/data';
+import { getPopperOptions } from '../feature_alert/utils';
 
 const knownTypes = arrayToHash(['left', 'right', 'top', 'bottom']);
 
-
 const Tooltip = (props) => {
+  const [isOpen, setOpen] = useState(false);
   const { type, text, children, ...cleanProps } = props;
   const selectedType = knownTypes[type] ? type : 'top';
-  const className = clsx(`c-tooltip c-tooltip-${selectedType}`, props.className);
+  const className = clsx('c-tooltip', props.className, {
+    'is-active': isOpen,
+  });
+
+  const handleOpen = useCallback((event) => {
+    event.stopPropagation();
+    setOpen(true);
+  }, [isOpen]);
+
+  const handleClose = useCallback((event) => {
+    event.stopPropagation();
+    setOpen(false);
+  }, [isOpen]);
+
+  const refElement = useRef(null);
+  const refTooltip = useRef(null);
+  const refArrow = useRef(null);
+
+  useEffect(() => {
```

Comment:
I think we should go with `useLayoutEffect` here and provide a proper teardown function